cpp/server/ct-server.cc

```diff
@@ -159,7 +160,8 @@ static bool ValidatePort(const char* flagname, int port) {
 static const bool port_dummy =
     RegisterFlagValidator(&FLAGS_port, &ValidatePort);
 
-static bool ValidateRead(const char* flagname, const string& path) {
+static bool ValidateRead(const char* flagname __attribute__((unused)),
```

Comment:
Does the compiler not complain that you used an unused variable?

On Wed, 22 Jul 2015 at 11:47 Al Cutter <notifications@github.com> wrote:

> In cpp/server/ct-server.cc
> <https://github.com/google/certificate-transparency/pull/842#discussion_r35200631>
> :
>
> > @@ -159,7 +160,8 @@ static bool ValidatePort(const char* flagname, int port) {
> >  static const bool port_dummy =
> >      RegisterFlagValidator(&FLAGS_port, &ValidatePort);
> >
> > -static bool ValidateRead(const char* flagname, const string& path) {
> > +static bool ValidateRead(const char* flagname __attribute__((unused)),
>
> __attribute__ induced blindness, removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/842/files#r35200631>
> .
>